examples/ts/count/src/store.ts

```diff
@@ -1,10 +1,10 @@
-import { init, RematchRootState } from '@rematch/core'
-import * as models from './models'
+import { init, RematchRootState, RematchDispatch } from '@rematch/core'
+import models from './models'
 
 export const store = init({
 	models,
 })
 
 export type Store = typeof store
-export type Dispatch = typeof store.dispatch
+export type Dispatch = RematchDispatch<typeof models>
```

Comment:
Is it required to create the dispatch method like this?

Reply:
No. This is  used in Model defination like `effects: (dispatch: Dispatch) => {}` to avoid recursive check.